components/script/dom/webgl/webgl2renderingcontext.rs

```diff
@@ -4310,6 +4310,12 @@ impl WebGL2RenderingContextMethods<crate::DomTypeHolder> for WebGL2RenderingCont
         varyings: Vec<DOMString>,
         bufferMode: u32,
     ) {
+        // Spec: If program was generated by a different WebGL2RenderingContext
+        // than this one, generates an INVALID_OPERATION error.
```

Comment:
Please use the specification step number here instead of "Spec: " and in other places.